packages/highlight/src/lib/themes/poimandres/poimandres.ts

```diff
@@ -0,0 +1,84 @@
+import {HighlightStyle, syntaxHighlighting} from '@codemirror/language';
+import {tags as t} from '@lezer/highlight';
+import {defineEditorTheme} from '../../core';
+
+export const palette = {
+  brightYellow: '#fffac2',
+  brightMint: '#5DE4c7',
+  lowerMint: '#5fb3a1',
+  blueishGreen: '#42675A',
+  lowerBlue: '#89ddff',
+  lightBlue: '#ADD7FF',
+  desaturatedBlue: '#91B4D5',
+  bluishGrayBrighter: '#7390AA',
+  hotRed: '#d0679d',
+  pink: '#f087bd',
+  gray: '#a6accd',
+  lighterGray: '#c0cad5',
+  darkerGray: '#767c9d',
+  bluishGray: '#506477',
+  focus: '#303340',
+  bg: '#1b1e28',
+  offWhite: '#e4f0fb',
+  selection: '#717cb4',
+  white: '#ffffff',
+  black: '#000000',
+};
+
+export const poimandres = [
+  syntaxHighlighting(
+    HighlightStyle.define([
+      {tag: t.definitionKeyword, color: palette.desaturatedBlue},
+      {tag: t.controlKeyword, color: palette.lowerMint},
+      {
+        tag: t.attributeName,
+        color: palette.desaturatedBlue,
+      },
+      {
+        tag: t.comment,
+        fontStyle: 'italic',
+      },
+      {tag: t.processingInstruction, color: palette.lighterGray},
+    ]),
```

Comment:
I would also add here the highlight of the `function` variable name declaration like VSCode does

![image](https://user-images.githubusercontent.com/37072694/178685381-6863396a-9bfa-4546-9374-b7805a8f10cd.png)

and the literal one style too 

![image](https://user-images.githubusercontent.com/37072694/178685459-9a395266-abb5-4204-9d27-4d5b89104c2f.png)

I think it could be something like that

```typescript
[
      // ... other tags
      {
        tag: [t.definition(t.function(t.variableName))],
        color: palette.lowerBlue,
      },
      {
        tag: [t.literal],
        color: palette.brightMint,
      },
]
```